1/Dockerfile.rhel7

```diff
@@ -47,14 +47,16 @@ ADD ./contrib/s2i /usr/libexec/s2i
 # NOTE: When adding new Jenkins plugin, you have to create the symlink for the
 # HPI file created by rpm to /opt/openshift/plugins folder.
 RUN mkdir -p /opt/openshift/plugins && \
-    ln -s /usr/lib64/jenkins/openshift-pipeline.hpi /opt/openshift/plugins/openshift-pipeline.hpi && \
-    ln -s /usr/lib64/jenkins/credentials.hpi /opt/openshift/plugins/credentials.hpi && \
+    ln -s /usr/lib64/jenkins/openshift-pipeline.hpi /opt/openshift/plugins/openshift-pipeline.jpi && \
+    ln -s /usr/lib64/jenkins/credentials.hpi /opt/openshift/plugins/credentials.jpi && \
     # Prefer the latest version of the credentials plugin over the bundled
     # version.
-    touch /opt/openshift/plugins/credentials.hpi.pinned && \
-    ln -s /usr/lib64/jenkins/durable-task.hpi /opt/openshift/plugins/durable-task.hpi && \
-    ln -s /usr/lib64/jenkins/kubernetes.hpi /opt/openshift/plugins/kubernetes.hpi && \
+    touch /opt/openshift/plugins/credentials.jpi.pinned && \
+    ln -s /usr/lib64/jenkins/durable-task.hpi /opt/openshift/plugins/durable-task.jpi && \
+    ln -s /usr/lib64/jenkins/kubernetes.hpi /opt/openshift/plugins/kubernetes.jpi && \
     /usr/local/bin/fix-permissions /opt/openshift && \
+    /usr/local/bin/fix-permissions /opt/openshift/plugins && \
+    chmod -R g+rw /opt/openshift/plugins/*.jpi && \
```

Comment:
yep, good catch on the link thing ... and yep, I got a find / exec chmod to work using your sample above.